support/models.py

```diff
@@ -179,6 +179,11 @@ class AccountDeletionRequest(models.Model):
         default=None,
     )
 
+    def __str__(self):
+        username = self.account.username if self.account else "Unknown User"
+        reason = f" - {self.reason}" if self.reason else ""
+        return f"Account Deletion Request: {username}{reason}"
```

Comment:
Only use username for creating str
```suggestion
    def __str__(self):
        username = self.account.username if self.account else "Unknown User"
        return username
```